README.md

```diff
@@ -106,15 +121,24 @@ This will pull all the events in the i3 file into `run*.evt*.json` which can be
 For now, it's easy to scale up using the command line. Multiple server instances can be run simultaneously and a separate submit file created for each one. To run `N` servers in parallel
 
 ```
-ls *.json | xargs -n1 -PN -I{} bash -c 'mkdir /path/to/json/{} && python -m skymap_scanner.server --startup-json-dir /path/to/json/{} --cache-dir /path/to/cache --output-dir /path/to/out --reco-algo RECO_ALGO --event-file /path/to/data/{} --broker BROKER_ADDRESS --auth-token AUTH_TOKEN --timeout-to-clients 300000 --timeout-from-clients 300000'
+export SKYSCAN_BROKER_AUTH=$(cat ~/skyscan-broker.token)  # obfuscated for security
+export SKYSCAN_MQ_TIMEOUT_TO_CLIENTS=300000
+export SKYSCAN_MQ_TIMEOUT_FROM_CLIENTS=300000
+ls *.json | xargs -n1 -PN -I{} bash -c 'mkdir /path/to/json/{} && python -m skymap_scanner.server --startup-json-dir /path/to/json/{} --cache-dir /path/to/cache --output-dir /path/to/out --reco-algo RECO_ALGO --event-file /path/to/data/{} --broker BROKER_ADDRESS --broker-auth BROKER_AUTH'
 ```
 
-Then, from sub-2 run `ls *.json |xargs -I{} bash -c 'sed "s/UID/{}/g" ../condor > /scratch/$USER/{}.condor'` using the template condor submit file below. Then you should be able to just run `ls /scratch/$USER/run*.condor|head -nN|xargs -I{} condor_submit {}`.
-
+Then, from sub-2 run `ls *.json |xargs -I{} bash -c 'sed "s/UID/{}/g" ../condor > /scratch/$USER/{}.condor'` using the template condor submit file below. Then you should be able to just run:
+```
+export SKYSCAN_BROKER_AUTH=$(cat ~/skyscan-broker.token)  # obfuscated for security
+export SKYSCAN_MQ_TIMEOUT_TO_CLIENTS=300000
+export SKYSCAN_MQ_TIMEOUT_FROM_CLIENTS=300000
+ls /scratch/$USER/run*.condor | head -nN | xargs -I{} condor_submit {}
+```
 ```
 executable = /bin/sh 
-arguments = /usr/local/icetray/env-shell.sh python -m skymap_scanner.client --broker BROKER_ADDRESS --auth-token AUTH_TOKEN --timeout-to-clients 3600 --timeout-from-clients 3600 --startup-json-dir .
+arguments = /usr/local/icetray/env-shell.sh python -m skymap_scanner.client --broker BROKER_ADDRESS --startup-json-dir .
 +SingularityImage = "/cvmfs/icecube.opensciencegrid.org/containers/realtime/skymap_scanner:x.y.z"
+getenv = SKYSCAN_*
```

Comment:
Maybe better to just add to the `environment` variable inside the submit file, instead of breaking them up into `export` and `getenv`.

Reply:
I'll check it out. I'll have to brush up on my condor

Reply:
I see what you mean now, I'll update the README